src/app/models/Product.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface Product {
+  uid: string; // This is an auto-generated field but we are including this for lisibility sake, will be set to the id of the document
+  images: Array<string>;
```

Comment:
```suggestion
  images: string[];
```
All the type in the interface must be TypeScript type and not mongo they will only be use in front and it's automatically interpretted